src/controls/SimpleLayersControl/index.js

```diff
@@ -100,48 +94,62 @@ class SimpleLayersControl extends React.PureComponent {
 						className="ptr-simple-layers-control-menu-content"
 						style={contentStyle}
 					>
-						{layers.map(layer =>
-							this.renderTile(layer, tileWidth, tileHeight, tileMargin)
+						{layerTemplates.map(layerTemplate =>
+							renderTile(layerTemplate, tileWidth, tileHeight, tileMargin)
 						)}
 					</div>
 				</div>
 			);
 		} else {
 			return null;
 		}
-	}
+	};
 
-	renderTile(layer, width, height, margin) {
-		let active = layer.key === this.props.activeLayer.key;
+	const renderTile = (layerTemplate, width, height, margin) => {
+		const active = layerTemplate.key === activeLayerTemplateKey;
 
-		let classes = classnames('ptr-simple-layers-control-tile', {
+		const classes = classnames('ptr-simple-layers-control-tile', {
 			active,
 		});
 
-		let style = {
+		const style = {
 			width: `${width}rem`,
 			height: `${height}rem`,
 			margin: `${margin}rem`,
 		};
-
-		if (layer.thumbnail) {
-			// TODO check type of thumbnail
-			// style.backgroundImage = `url(${require('./img/' + layer.thumbnail + '.png')})`;
+		let previewParam = {};
+		if (layerTemplate?.data?.thumbnail) {
+			previewParam['src'] =
+				layerTemplate.data.thumbnail in images
+					? images[layerTemplate.data.thumbnail]
+					: images.noPreview;
+			previewParam['alt'] = layerTemplate.data.thumbnail;
 		}
```

Comment:
There is no else for case if layerTemplate has not thumbnail. 

Thre should bo some placeholder?

Reply:
https://github.com/gisat-panther/ptr-maps/pull/84/commits/3ec8eb7775634c6957d146e0f68dde1366fef4b5